projects/pffft/pffft_fuzzer.cc

```diff
@@ -0,0 +1,70 @@
+// Copyright 2018 Google Inc.
```

Comment:
We typically [recommend](https://github.com/google/oss-fuzz/blob/master/docs/ideal_integration.md) that the fuzz target is maintained by the project owners and in their repo, at least in the long run. 
Did you talk to them?

Reply:
Thanks for asking. Yes, the author is aware. Note that the last PFFFT upstream commit is from 2016 and it's unlikely that the interface of something well-established as the FFT changes. However, I understand that it's better to move the fuzzer upstream.

To avoid delays, is it possible to accept this change and then I will create a pull request upstream to move the fuzzer in there? Then, when merged, I will update oss-fuzz by removing the fuzzer source code and adapting build.sh.

Reply:
sure. I just want to have this done eventually, at least if there is any movement in the project (like bugs found and fixed)